### ISSUE TITLE
German translation update

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#200]
```
I reviewed the german translation and adjusted several strings.
See: http://andors.techby2guys.com/viewtopic.php?p=2137#p2137

File: http://andors.techby2guys.com/download/file.php?id=46
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 11 Jun 2011 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
In the same thread Bomber also updated his translation.
```

Original comment by `SamuelPl...@gmail.com` on 15 Jun 2011 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
I've made an update on my post to show an actual translation status: 
http://andors.techby2guys.com/viewtopic.php?f=6&t=329&p=1385#p1385
```

Original comment by `speedbom...@googlemail.com` on 16 Jun 2011 at 3:09

**Comment by GoogleCodeExporter** [comment]
```
Committed in r133.

Thank you for the great German translation!
```

Original comment by `oskar.wi...@gmail.com` on 22 Jun 2011 at 8:23
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:53
- Changed state: **Done**
